### PR TITLE
Closed #267

### LINKED ISSUE #267: Let compatibility module loaders know the supported version
If the detected dependency version doesn't match the supported one(s), a warning will be logged.
~~It's configurable for servers to disable compat modules with no exact dependency match.~~

### PR DESCRIPTION
**Pull request by CraftedMods**

### CODE REVIEW

File: src/main/java/pvpmode/api/common/compatibility/IdentifierCompatibilityModuleLoader.java [hunk 1]
```diff
@@ -59,4 +61,39 @@ public String getInternalModuleName ()
         return StringUtils.join (identifiers);
     }
 
+    protected boolean isVersionSupported (String identifier, String version)
+    {
+        return true;
+    };
```
Comment (Mahtaran) [inline src/main/java/pvpmode/api/common/compatibility/IdentifierCompatibilityModuleLoader.java]: typo

Comment (CraftedMods) [reply src/main/java/pvpmode/api/common/compatibility/IdentifierCompatibilityModuleLoader.java]: Where's it?

Comment (Mahtaran) [reply src/main/java/pvpmode/api/common/compatibility/IdentifierCompatibilityModuleLoader.java]: Semicolon after the curly bracket.

Comment (CraftedMods) [reply src/main/java/pvpmode/api/common/compatibility/IdentifierCompatibilityModuleLoader.java]: Oh. How did it get there?

**Pull request merged**